docs/Config-Specification.rst

```diff
@@ -897,17 +897,17 @@ node_templates:
 |             pxe:                   |               interface: eth15                | |   *interface* - Reference to interface label defined in the `interfaces:`_     |          |
 |                 - switch:          |               rename: true                    |                elements.                                                         |          |
 |                   interface:       |               ports:                          | |   *rename* - Value (true/false) to control whether client node interfaces will |          |
```

Comment:
spelling 'inerface'
can we change 'iface or DEVICE value' to be: 'iface (Ubuntu) or DEVICE (Red Hat) value'  Unless you're familiar with both nomenclatures, this is confusing as is.

Reply:
Fixed both in eb4f677